roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -65,6 +65,6 @@
 
 - name: Set docker version to be installed
   set_fact:
-    docker_version: "{{ '1.9.1' }}"
+    docker_version: "{{ '1.10.3' }}"
```

Comment:
that said, my preference would be to implement gte, but I've yet to come up with a clean way to do that. I guess we could do bounds checking in ansible, if the to-be-installed version fits within those bounds then just run yum without a version. If to-be is outside of those bounds then set version to the upper bound.